xbmc/cores/VideoPlayer/VideoRenderers/windows/RendererBase.h

```diff
@@ -188,4 +189,5 @@ class CRendererBase
   DXGI_HDR_METADATA_HDR10 m_lastHdr10 = {};
   HDR_TYPE m_HdrType = HDR_TYPE::HDR_NONE_SDR;
   bool m_AutoSwitchHDR = false;
+  std::string m_renderMethodName{"NA"};
```

Comment:
Probably don't worth initialize (empty string is fine)

Reply:
sure